doc/dev/logs.rst

```diff
@@ -9,8 +9,6 @@ Dout has several log faculties, which can be set at various log
 levels using the configuration management system. So it is possible to enable
 debugging just for the messenger, by setting debug_ms to 10, for example.
 
-Dout is implemented mainly in common/DoutStreambuf.cc
```

Comment:
Which file was `DoutStreambuf.cc` swapped to?

Reply:
Please see: https://github.com/ceph/ceph/blob/master/src/common/debug.h